src/BloomExe/CollectionChoosing/OpenCreateCloneControl.cs

```diff
@@ -79,7 +79,7 @@ private void OnLoad(object sender, EventArgs e)
 
 			foreach (Control control in tableLayoutPanel2.Controls)
 			{
-				if (control.Tag == "sendreceive")
+				if (control.Tag != null && control.Tag.ToString() == "sendreceive")
 					control.Visible = Settings.Default.ShowSendReceive;
```

Comment:
That particular bit? Nothing. My compiler was giving the complaint I mentioned in the PR description above, so I fixed it.
No real change in logic, just safer and clearer.